docs/source/resource_files.rst

```diff
@@ -400,7 +375,7 @@ Below is a list of the init files that are packaged with PROJ.
     ========    ================================================================
 
 
-The defaults file
+The defaults file (obsolete)
```

Comment:
This could just as well be removed, I think. Very few has used it when it was available so I don't believe it will be missed from the docs.

Reply:
yes was also considering just stripping it out. Done